### PR TITLE
New Routes Added

### PR DESCRIPTION
**Pull request by mdhishaamakhtar**
I have added the route for the participants to post the URL of their GitHub repositories. The other two are under development.

### CODE REVIEW

File: routes/postgit.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+const verify = require("./verifyToken");
+const Users = require("../model/User");
+
+router.post("/", verify, async (req, res) => {
+  Users.findByIdAndUpdate(req.user,{ $push: { github: req.body.github } } , err => {
+    if (err) {
+      console.error(err);
+    }
+  });
+  res.send("Pushed the URL of your repository");
```
Comment (L04DB4L4NC3R) [inline routes/postgit.js]: This should be inside the `findByIdAndUpdate` function

File: routes/getusers.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+const router = require("express").Router();
+const verify = require("./verifyToken");
+const Users = require("../model/User");
+
+router.get("/",verify, (req, res) => {
+  Users.find({})
+    .select("-password")
+    .then(users => {
+      res.send(users);
+    });
```
Comment (L04DB4L4NC3R) [inline routes/getusers.js:14]: Add a `.catch` to handle errors here

**Review by L04DB4L4NC3R — CHANGES_REQUESTED** [review]

File: routes/getusers.js (hunk 2 above)
Comment (mdhishaamakhtar) [reply routes/getusers.js:14]: I have resolved it.

File: routes/postgit.js (hunk 1 above)
Comment (mdhishaamakhtar) [reply routes/postgit.js]: I have resolved it.

**Pull request merged**